File: Week 6/mario.py
from cs50 import get_int

# n = get_int("Height: ")
# while True:
#     n = get_int("Height: ")
#     if n > 0:
#         break

# for i in range(n):
#     print("#")

def main():
    height = get_height()
    for i in range(height):
        print("#")

def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0:
            break
    return n

main()

File: Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);

int main(void)
{
    // Prompt the user for the pyramid's height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    // accept only numbers 1 through 8
    while (n <= 0 || n >= 9);

    // Print a pyramid of this height
    for (int i = 0; i < n; i++)
    {
        // Print row of bricks in the correct amounts compared to eachother
        print_row(n - i - 1, i + 1);
    }
}

void print_row(int spaces, int bricks)
{
    // Print some spaces
    for (int i = 0; i < spaces; i++)
        printf(" ");
    {
        for (int i = 0; i < bricks; i++)
            // Print some bricks
            printf("#");
    }
    // print next line
    printf("\n");
}
